packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -83,10 +93,144 @@ test.describe('`Pie Modal is closed`', () => {
                 },
             );
 
+            // Act
             await page.locator('#dialog').click();
 
+            // Assert
             expect(messages).toHaveLength(1);
         });
     });
 });
 
+test.describe('Pie Modal `isDismissible`', () => {
+    test.describe('when `isDismissible` is truthy', () => {
+        test('should contain a close button so the modal can be dismissed', async ({ mount }) => {
+            // Arrange
+            const component = await mount(
+                PieModal,
+                {
+                    props: {
+                        isOpen: true,
+                        isDismissible: true,
+                    },
+                },
+            );
+
+            // Act & Assert
+            await expect(component.locator('.c-modal-closeBtn')).toBeVisible();
```

Comment:
I like the test, but we should probably hook onto a data-test-id rather than a class

Reply:
fixed.

Reply:
I still see this using a class, did the changes to use a `data-test-id` get pushed?

Reply:
The files is outdated so will show the old code still.